certbot-apache/certbot_apache/_internal/configurator.py

```diff
@@ -2284,23 +2285,7 @@ def _get_redirect_config_str(self, ssl_vhost):
             rewrite_rule_args = constants.REWRITE_HTTPS_ARGS_WITH_END
         else:
             rewrite_rule_args = constants.REWRITE_HTTPS_ARGS
-
-        return ("<VirtualHost %s>\n"
-                "%s \n"
-                "%s \n"
-                "ServerSignature Off\n"
-                "\n"
-                "RewriteEngine On\n"
-                "RewriteRule %s\n"
-                "\n"
-                "ErrorLog %s/redirect.error.log\n"
-                "LogLevel warn\n"
-                "</VirtualHost>\n"
-                % (" ".join(str(addr) for
-                            addr in self._get_proposed_addrs(ssl_vhost)),
-                   servername, serveralias,
-                   " ".join(rewrite_rule_args),
-                   self.options.logs_root))
+        return rewrite_rule_args
```

Comment:
What happened here?

I think the test suite is failing due to this change.

Reply:
I meant to use f-strings here, but pushed before getting to it. Will do it now :) Thank you!